api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -175,12 +176,13 @@ func (clusterVirtualMachineImage *ClusterVirtualMachineImage) SetConditions(cond
 // +kubebuilder:resource:scope=Cluster,shortName=cvmi;cvmimage;clustervmimage
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="ContentSourceName",type="string",JSONPath=".spec.providerRef.name"
-// +kubebuilder:printcolumn:name="ContentLibraryName",type="string",JSONPath=".status.contentLibraryRef.name"
+// +kubebuilder:printcolumn:name="Provider-Name",type="string",JSONPath=".spec.providerRef.name"
```

Comment:
Just curious on what can be the provider-name of an VM image, any example?
Also is the new printcolumn naming pattern (separated by -) consistent with other APIs?

Reply:
@ericvmw "provider-name" is the name of content-library-item (or cluster-content-library-item).  I used this name here to match the JSON path name of the resource field.
There is no strict convention on printcolumn naming patterns from K8s (it uses both "-" and space for the internal resources). I chose "-" as suggested by BV' [comment](https://github.com/vmware-tanzu/vm-operator-api/pull/109#issuecomment-1289678131) and updated all our APIs in this PR to keep the consistency.

Reply:
I see. So this is the resource name of the corresponding content-library-item or cluster-content-library-item, and the `image-name` in the status is the name of the item / image from vSphere.